From 58b0d4b02eff9507d6105a2d724584e02a8744df Mon Sep 17 00:00:00 2001
From: Julien Boibessot <julien.boibessot@armadeus.com>
Date: Fri, 17 Aug 2018 14:57:51 +0000
Subject: [PATCH] Increase firmware boot wait timeout, otherwise, on OPOS6UL,
 one need to do: 	# modprobe -r brcmfmac 	# modprobe brcmfmac to have a
 working Wi-Fi.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

---
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
index 1037df7..f59fe36 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
@@ -299,7 +299,7 @@ struct rte_console {
  * max out at a 1 second (1000000us).
  */
 #undef PMU_MAX_TRANSITION_DLY
-#define PMU_MAX_TRANSITION_DLY 1000000
+#define PMU_MAX_TRANSITION_DLY 5000000
 
 /* Value for ChipClockCSR during initial setup */
 #define BRCMF_INIT_CLKCTL1	(SBSDIO_FORCE_HW_CLKREQ_OFF |	\
